vmdb/db/migrate/20141029005602_create_configuration_managers.rb

```diff
@@ -0,0 +1,75 @@
+class CreateConfigurationManagers < ActiveRecord::Migration
+  def up
+    create_table :configuration_managers do |t|
+      t.belongs_to :provider, :type => :bigint
+      t.string     :type
+      t.timestamps
+    end
+    add_index :configuration_managers, :provider_id
+
+    create_table :configuration_profiles do |t|
+      t.string     :type
+      t.string     :name
+      t.string     :description
+      t.belongs_to :operating_system_flavor, :type => :bigint
+      t.belongs_to :configuration_manager,   :type => :bigint
+      t.string     :manager_ref
```

Comment:
nice...we've had ems_ref for so long I'm just drawn to it.  This is much better as we move forward.